pkg/controllers/node/termination/terminator/events/events.go

```diff
@@ -43,3 +53,13 @@ func NodeFailedToDrain(node *v1.Node, err error) events.Event {
 		DedupeValues:   []string{node.Name},
 	}
 }
+
+func NodeTerminationGracePeriod(node *v1.Node, expirationTime string) events.Event {
+	return events.Event{
```

Comment:
IMO, we should make an events.k8s.io/v1 [Event](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/event-v1/) with:
- the Node being tainted as the `regarding` object
- the NodePool [for the NodeClaim for the Node] as the `related` object

K8s' API lets us identify two objects, and there are two relevant things, so let's mention both